printNames.js

```diff
@@ -0,0 +1,13 @@
+const f = require("fs");
+const emptyfunc =()=>{
```

Comment:
Optional: you can place this inside the emptyfunc location, just use (err)=>{} or your empty function.